lib/ui/onboard/widgets/operator_biometrics_capture_view.dart

```diff
@@ -148,9 +150,29 @@ class _OperatorBiometricsCaptureState
           ),
         ));
   }
-
   @override
   Widget build(BuildContext context) {
+    globalProvider = context.watch<GlobalProvider>();
+    biometricCaptureControlProvider = context.watch<BiometricCaptureControlProvider>();
+    final authProvider = context.watch<AuthProvider>();
+    final appLocalizations = AppLocalizations.of(context)!;
+
+    String getBiometricTitle() {
+      if (authProvider.isOfficer) {
+        return globalProvider.onboardingProcessName == "Onboarding"
+            ? appLocalizations.supervisors_biometric_onboard
+            : appLocalizations.supervisors_biometric_update;
+      } else if (authProvider.isOperator) {
+        return globalProvider.onboardingProcessName == "Onboarding"
+            ? appLocalizations.onboard_operator_biomterics
+            : appLocalizations.update_operator_biomterics;
+      } else {
+        return globalProvider.onboardingProcessName == "Onboarding"
+            ? appLocalizations.supervisors_biometric_onboard
+            : appLocalizations.supervisors_biometric_update;
+      }
+    }
```

Comment:
this can be reduced to only one if condition

if (authProvider.isOperator) {
        return globalProvider.onboardingProcessName == "Onboarding"
            ? appLocalizations.onboard_operator_biomterics
            : appLocalizations.update_operator_biomterics;
}

return globalProvider.onboardingProcessName == "Onboarding"
            ? appLocalizations.supervisors_biometric_onboard
            : appLocalizations.supervisors_biometric_update;